packages/keystone/logging/getKeystonePinoOptions.js

```diff
@@ -14,6 +15,12 @@ function getKeystonePinoOptions () {
         customProps: (req, res) => {
             return getReqLoggerContext(req)
         },
+        serializers: {
+            res: (res) => {
+                const errors = res.raw.req.errors
```

Comment:
Should it be wrapped with get or optional chain in case of no errors or smth like this?

Reply:
Yup, sounds reasonable